maxim/logger/livekit/realtime_session.py

```
@@ -130,19 +131,56 @@ def handle_off(self):
+            offset = chunk_end
+            current_index += 1
+        
+        session_info.conversation_buffer_index = current_index
+    else:
+        # Buffer is small enough, add as single attachment
+        get_maxim_logger().session_add_attachment(
+            session_id,
+            FileDataAttachment(
+                data=pcm16_to_wav_bytes(buffer_data),
+                tags={"attach-to": "input"},
+                name=f"Conversation",
```

Ruff (0.14.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)